go.mod

```diff
@@ -2,17 +2,15 @@ module github.com/stashapp/stash
 
 require (
 	github.com/99designs/gqlgen v0.9.0
-	github.com/PuerkitoBio/goquery v1.5.0
 	github.com/antchfx/htmlquery v1.2.3
-	github.com/bmatcuk/doublestar v1.1.5
+	github.com/bmatcuk/doublestar v1.3.1
 	github.com/disintegration/imaging v1.6.0
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/gobuffalo/packr/v2 v2.0.2
 	github.com/golang-migrate/migrate/v4 v4.3.1
 	github.com/gorilla/sessions v1.2.0
 	github.com/gorilla/websocket v1.4.0
 	github.com/h2non/filetype v1.0.8
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
```

Comment:
Can we leave this in for now. I think we'll want to change the packr dependency to get rid of this issue altogether, but this is required for windows builds currently, and also exists in the vendor directory.

Reply:
Yeah i keep forgetting that, it gets removed by the `go mod tidy`.
The cross-compile script seems to have made a windows binary though. Is it only needed when compiling from windows ?

Reply:
It seems a be a wrinkle in go. The tidy command doesn't take into account cross-platform dependencies, but the vendor command must do.